projects/1.20/assets/doggy-talents-next/doggytalents/lang/zh_cn.json

```
@@ -0,0 +1,824 @@
+    "item.doggytalents.locator_orb_ka.description": "火焰的宝珠",
+    "item.doggytalents.locator_orb_sui": "水之玉",
+    "item.doggytalents.locator_orb_sui.description": "水流的宝珠",
+    "item.doggytalents.locator_orb_moku": "木之玉",
+    "item.doggytalents.locator_orb_moku.description": "植物之玉",
+    "item.doggytalents.locator_orb_dyable": "可染色定位珠",
+    "item.doggytalents.striped_scarf": "条纹围巾",
+    "item.doggytalents.head_band_blank": "头巾",
+    "item.doggytalents.head_band_mystery": "头巾 - 神秘",
+    "item.doggytalents.head_band_mystery.description": "那个狼系男孩是谁？",
+    "item.doggytalents.head_band_highhh": "头巾 - 嗨嗨嗨~",
+    "item.doggytalents.head_band_highhh.description": "开发者最爱的颜文字",
+
+    "item.doggytalents.gender_bone": "性别转换零食",
+    "item.doggytalents.variant_changer": "变种切换器",
+    "item.doggytalents.variant_changer.description": "变种：%s",
+
+    "item.doggytalents.superdog_suit": "超级狗套装",
+    "item.doggytalents.flying_cape": "飞行斗篷",
+    "item.doggytalents.bat_wings": "蝙蝠翅膀",
+    "item.doggytalents.crow_wings": "乌鸦翅膀",
+    "item.doggytalents.angel_wings": "天使翅膀",
+    "item.doggytalents.angel_halo": "天使光环",
+    "item.doggytalents.goose": "MrBlueYeti的鹅",
+    "item.doggytalents.goose.description": "嘎！嘎！",
+    "item.doggytalents.midi_keyboard": "MIDI键盘",
+
+    "item.doggytalents.divine_retribution": "神圣惩戒",
+    "item.doggytalents.divine_retribution.description": "\"好烫好烫！快从我背上拿下来！\" ——普通狗狗说",
+    "item.doggytalents.divine_retribution.description2": "\"你会习惯的。\" ——Kumi回答",
+    "item.doggytalents.soul_reflector": "灵魂反射器",
+
+    "item.doggytalents.rice_grains": "稻谷",
+    "block.doggytalents.rice_crop.description": "右键耕地种植水稻。潜行+右键砂轮去壳",
+    "item.doggytalents.uncooked_rice": "生米",
+    "item.doggytalents.uncooked_rice_bowl": "碗装生米",
+    "item.doggytalents.uncooked_rice_bowl.description": "放进熔炉里烤吧！",
+    "item.doggytalents.rice_wheat": "水稻",
+    "item.doggytalents.rice_wheat.description": "用于制作米粒",
+    "block.doggytalents.rice_crop": "水稻作物",
+    "block.doggytalents.rice_mill": "碾米机",
+    "item.doggytalents.koji": "麴菌",
+    "item.doggytalents.soy_beans": "大豆",
+    "item.doggytalents.soy_pods": "豆荚",
+    "item.doggytalents.soy_pods.description": "右键剥开豆荚",
+    "block.doggytalents.soy_crop": "大豆作物",
+    "item.doggytalents.soy_beans_dried": "干大豆",
+    "item.doggytalents.soy_pods_dried": "干豆荚",
+    "item.doggytalents.soy_pods_dried.description": "右键剥开豆荚",
+    "item.doggytalents.edamame": "毛豆",
+    "item.doggytalents.edamame.description": "又名盐水煮嫩豆。你和狗狗都能吃！\n但记得先剥壳！\n右键剥开豆荚",
+    "item.doggytalents.edamame_unpodded": "去壳毛豆",
+    "item.doggytalents.edamame_unpodded.description": "我开动了！",
+    "item.doggytalents.miso_paste": "味噌酱",
+    "item.doggytalents.miso_paste.description": "咸香！适合做味噌汤！",
+    "item.doggytalents.miso_soup": "味噌汤",
+    "item.doggytalents.miso_soup.description": "我开动了！",
+    "item.doggytalents.soy_milk": "豆浆",
+    "item.doggytalents.tofu": "豆腐",
+    "item.doggytalents.tofu.description": "素食主义者的最爱！",
```

```suggestion
    "item.doggytalents.tofu.description": "适合素食者！",
```
”Suitable for Vegans!“，请按照原文一一对应。